src/posterior_gp/sequential_conditioning.jl

```diff
@@ -0,0 +1,47 @@
+
+function posterior(fx::FiniteGP{<:PosteriorGP}, y::AbstractVector{<:Real})
+    m2 = mean(fx.f.prior, fx.x)
+    y2_m2 = y - m2
+    C11 = fx.f.data.C.U' * fx.f.data.C.U
+    U11 = fx.f.data.C.U
+    C12 = cov(fx.f.prior, fx.f.data.x, fx.x)
+    C22 = cov(fx.f.prior, fx.x) + fx.Σy
+    U = update_chol(U11, C12, C22)
+
+    #TODO: Better way to get Cholesky struct from the decomposition.
+    chol = cholesky(U' * U)
```

Comment:
```suggestion
    C = Cholesky(U, 'U', 0)
```
I think this should do it - it just accesses the `Cholesky` constructor directly.